components/Dashboard/TransactionTable.vue

```diff
@@ -64,19 +273,74 @@ export default {
     await axios
       .get("https://pwcdemo-1c4d3.firebaseio.com/users.json")
       .then(response => {
-        (this.loading = false), (this.facts = Object.values(response.data));
+        const postArray = [];
+        for (const key in response.data) {
+          postArray.push({ ...response.data[key], id: key });
+        }
+        (this.loading = false), (this.facts = postArray);
```

Comment:
this is anti-pattern and hard to read, why don't you expand it to:
```
this.loading = false
this.facts = postArray
```

Reply:
My compiler was have some problem. When i save this page, it does not save. sorry...
I solved this problem. You can see code file.